cicd/gitlab/dot.gitlab-ci.yml

```diff
@@ -0,0 +1,72 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# WORKFLOW RULES -- WHEN TO RUN PIPELINES
+
+# workflow:rules -- using template instead
+#   https://docs.gitlab.com/ee/ci/yaml/#workflow
+# https://docs.gitlab.com/ee/ci/yaml/workflow.html#switch-between-branch-pipelines-and-merge-request-pipelines
+#workflow:
+#  rules:
+#    - if: $CI_PIPELINE_SOURCE == "merge_request_event"
+#    - if: $CI_COMMIT_BRANCH && $CI_OPEN_MERGE_REQUESTS
+#      when: never
+#    # - if: $CI_COMMIT_BRANCH
+
+
+variables:
+  RUN_DIR: ./cicd/runtime
+
+stages:
+  - debug
+  - test
+
+# In-house stuff
+
+#   CI/CD debugging
+debug:gitlab_env:
+  # comment out when section to run
+  when: manual
+  stage: debug
+  # using sub-pipeline from ./parts
+  trigger:
+    strategy: depend
+    include:
+    - local: /cicd/gitlab/parts/debug.gitlab-ci.yml
```

Comment:
Based on the content, I assume this is just to check everything is configured properly?

Reply:
it's more about checking env variable values